src/backend/interfaces/REST/go.server/middlewares/authentication.go

```diff
@@ -25,9 +28,14 @@ func BasicAuthFromCache(cache *redis.Client, realm string) gin.HandlerFunc {
 		// Get provided auth headers
 		var user_id, access_token string
 		var ok bool
-		if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
-			kickUnauthorizedRequest(c, realm)
+
+		//Try auth-scheme 'Bearer' then 'Basic'
+		if user_id, access_token, ok = BearerAuth(c.Request); !ok {
+			if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
```

Comment:
so we must use `Basic` instead of `Bearer`. WDYT @gdchamal ?

(I've no idea why we start with "Bearer")